saltstack/base/salt/prerequisites/packages.sls

```diff
@@ -64,6 +64,7 @@ packages_install:
       - sysstat
       - goaccess
       - httpd-tools
+      - cryptsetup
```

Comment:
I think we only need this for RHEL8 (and later for any subsequent RHEL version meant for AWS GovCloud). So we should like put this inside `{% if pillar['OS'] == 'redhat8' %}` (line 51).